src/components/TopBar.tsx

```diff
@@ -135,6 +161,13 @@ export const TopBar: React.SFC<Props> = (props) => (
             ))
           : null}
       </BarProgressContainer>
+      <CollapsedProgressContainer>
```

Comment:
maybe we should actually hide the whole container if we don't have a progress?

Reply:
hmm, think the naming might be confusing, the CollapsedProgressContainer is only showing the current step in the process

Reply:
Oh I see what you mean!